yoke/assets/joypad/base.js

```diff
@@ -874,28 +875,35 @@ function loadPad(filename) {
     link.onload = function() {
         document.getElementById('joypad').style.display = 'grid';
         if (window.CSS && CSS.supports('display', 'grid')) {
-            warningDiv.style.display = 'none';
-            if (PRESSURE_DETECTION_ENABLED == false || joypad == 0 || joypad == 1 || joypad == null) { // no touch force detection capability
-                joypad = new Joypad();
-            } else { // possible force detection capability
+            document.getElementById('warning').style.display = 'none';
+            if (pressureSensitive) { // touch force detection capability
                 Pedal.prototype.onTouchMove = Pedal.prototype.onTouchMoveForce;
                 AnalogButton.prototype.processTouches = AnalogButton.prototype.processTouchesForce;
                 Joystick.prototype.checkThumbButton = Joystick.prototype.checkThumbButtonForce;
                 joypad = new Joypad();
+            } else { // no touch force detection capability
```

Comment:
I think this else branch could be eliminated now.

```diff
-   joypad = new Joypad();
- } else {
-   joypad = new Joypad();
- }
+ joypad = new Joypad();
```

Reply:
Oops. d5beb667899c3cc7b6e35f5a97ea884bc85a9b87